File: async-scala-client/src/main/scala/io/swagger/client/model/NRef_refNF.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class NRef_refNF (
  stateCd: StateEnum,  // State code of fiscal Document creator
yymm: String,  // year and month of fiscal document creation
federalTaxId: String,  // fiscal document creator federalTaxId
serie: String,  // fiscal document serie
number: String  // fiscal document number
)

File: javascript-closure-angular-client/API/Client/NRefRefNF.js
goog.provide('API.Client.NRef_refNF');

/**
 * @record
 */
API.Client.NRefRefNF = function() {}

/**
 * State code of fiscal Document creator
 * @type {!API.Client.StateEnum}
 * @export
 */
API.Client.NRefRefNF.prototype.stateCd;

/**
 * year and month of fiscal document creation
 * @type {!string}
 * @export
 */
API.Client.NRefRefNF.prototype.yymm;

/**
 * fiscal document creator federalTaxId
 * @type {!string}
 * @export
 */
API.Client.NRefRefNF.prototype.federalTaxId;

/**
 * fiscal document serie
 * @type {!string}
 * @export
 */
API.Client.NRefRefNF.prototype.serie;

/**
 * fiscal document number
 * @type {!string}
 * @export
 */
API.Client.NRefRefNF.prototype.number;

